src/libxtp/gaussianwriter.cc

```diff
@@ -0,0 +1,307 @@
+#include "votca/xtp/gaussianwriter.h"
+#include "votca/xtp/basisset.h"
+#include <boost/algorithm/string.hpp>
+#include <boost/format.hpp>
+#include <sstream>
+#include <votca/tools/eigenio_matrixmarket.h>
+
+namespace votca {
+namespace xtp {
+
+/*
+ * This function converts VOTCA's L enum to the gaussian equivalent.
+ * Gaussian uses a minus sign to indicate spherical shells.
+ */
+Index GaussianWriter::toGaussianL(L l) const {
+  switch (l) {
+    case L::S:
+      return 0;
+    case L::P:
+      return 1;
+    default:
+      return -1 * static_cast<Index>(l);
```

Comment:
why do you have to switch for S and P?

Reply:
Because a -1 for the p shell translates to an sp shell in gaussian and -0 doesn't make much sense.

Reply:
okay